app/localization.js

```diff
@@ -84,40 +79,32 @@ Localizer.prototype = {
 
     // Retrieve localised text
     get: function (id) {
-        if (typeof this.dictionary !== 'undefined' && this.dictionary[id]) {
-            return this.dictionary[id];
-        } else {
-            return id;
-        }
+        return (typeof this.dictionary !== 'undefined' && this.dictionary[id])
+            ? this.dictionary[id]
+            : id;
     },
 
     // Traverses the DOM and translates relevant fields
     // See https://html.spec.whatwg.org/multipage/dom.html#attr-translate
     translateDOM: function () {
-        var self = this;
+        const self = this;
         function process(elem, enabled) {
             function isAnyOf(searchElement, items) {
                 return items.indexOf(searchElement) !== -1;
             }
 
             function translateAttribute(elem, attr) {
-                var str = elem.getAttribute(attr);
-                str = self.get(str);
+                const str = self.get(elem.getAttribute(attr));
                 elem.setAttribute(attr, str);
             }
 
             function translateTextNode(node) {
-                var str = node.data.trim();
-                str = self.get(str);
+                const str = self.get(node.data.trim());
                 node.data = str;
             }
 
             if (elem.hasAttribute("translate")) {
-                if (isAnyOf(elem.getAttribute("translate"), ["", "yes"])) {
-                    enabled = true;
-                } else if (isAnyOf(elem.getAttribute("translate"), ["no"])) {
-                    enabled = false;
-                }
+                enabled = isAnyOf(elem.getAttribute("translate"), ["", "yes"]);
```

Comment:
This is a behavioural change. Does this still follow the spec?

Reply:
I can't see any difference between
```
if (var1 = true) {
  var2 = true
} else {
  var2 = false
}
```

or `var1 = var2`

unless you want to be really strict with typing, in which case I'd still do `var1 = !!var2`